src/main/java/org/mule/extension/file/common/api/matcher/FileMatcher.java

```diff
@@ -105,6 +107,10 @@
   private Long maxSize;
 
 
+  private PredicateType predicateType = LOCAL_FILE_SYSTEM;
+
+  private boolean caseSensitive = true;
```

Comment:
I think you should have a public static final with the default value and use it in all the classes that need to set the default value. If you don't do that, you have the risk that changing a value in the future in one class by setting different values in each class.